src/peft/tuners/loha/config.py

```diff
@@ -86,6 +107,18 @@ class LoHaConfig(LycorisConfig):
             )
         },
     )
+    use_khatri_rao: bool = field(
+        default=False,
```

Comment:
Note for the future: If we can observe that the numerical difference of using Khatri-Rao is negligible and it is strictly better memory-wise, I wouldn't be opposed to make it the default.